test/performance/io/format_fastq_benchmark.cpp

```diff
@@ -0,0 +1,256 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
+// @author: Clemens Cords <clemens.cords@fu-berlin.de>
+
+#include <benchmark/benchmark.h>
+#include <cstring>
+#include <fstream>
+
+#include <seqan3/test/seqan2.hpp>
+#if SEQAN3_HAS_SEQAN2
+    #include <seqan/seq_io.h>
```

Comment:
I know other benchmarks indent here, but we normally don't do that. And I think the other benchmarks do it wrong for includes.

```suggestion
#include <seqan/seq_io.h>
```